### ISSUE TITLE
Update LR Settings button not working

### ISSUE THREAD

**Issue by tejennings** [maratbn/LoginRequirePress#5]
Is this because of wordpress version 4.4 launch?

**Comment by maratbn** [comment]
Likely not.

In what way is it "not working" for you?

**Comment by tejennings** [comment]
I tick a post to select it in the list, then click the LR settings button to save it, but when I look at the list afterwards the post is unticked again and obviously doesn't force the post visitor to the login page like it's supposed to. Other than plugin updates & the wordpress 4.4 update we haven't made any changes to the site.

**Comment by maratbn** [comment]
I just retested again on WordPress 4.4 and still not seeing any issues.

Are these regular pages and posts or some other post type?  Look under the 'Post Type' column of the table.

There is a possibility that there is a conflict with another plugin installed on your site.  Would you mind to share the list of plugins you have installed on here?

If it is another plugin, the way to troubleshoot that would be to deactivate the plugins one by one, and try locking a post in the UI after each deactivation.  If you are finally able to lock a post after disabling enough plugins, it means that one or more of those plugins were interfering.

If you don't want to do this on your production site, which I recommend you do not, I suggest you set up a separate test site with WordPress 4.4, LoginRequirePress, and a bunch of dummy pages and posts.  Start by testing the LoginRequirePress UI functionality on the bare site, and then start installing the plugins you use one by one until the UI stops working.

If you can look at your server's error log that would also be helpful, as there might be a PHP error emitted when you're trying to lock a post that would give me a clue as to what is going on.

-- Marat

**Comment by tejennings** [comment]
Thanks for re-testing, these are just regular posts & pages under the 'post type' column of the table. I've attached a list of our active plugins too, and will also try the plugins on a test site next. We didn't have an error log activated for the site, so I've had the hosting company activate one today and will send you a copy once it's had 24hrs to run. Thanks again for looking into this.  
[plugins.docx](https://github.com/maratbn/LoginRequirePress/files/74334/plugins.docx)

**Comment by maratbn** [comment]
Yeah that's a lot of plugins you guys have.  Did you guys see any difference after disabling any of them?  Any interesting errors in the log?
